viral/grosmark_analysis.py

```
@@ -137,6 +139,10 @@ def get_place_cells(
     sigma_bins = sigma_cm / config.bin_size  # Convert to bin units
 
     n_shuffles = 2000
+    if n_shuffles < 2000:
+        warnings.warn(
```

Ruff (0.13.3)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)